canonicalwebteam/blog/common_view_logic.py

```diff
@@ -5,15 +5,15 @@
 group_cache = {}
 
 
-def get_index_context(page_param, articles, total_pages):
+def get_index_context(page_param, articles, total_pages, featured_articles=[]):
     """
     Build the content for the index page
     :param page_param: String or int for index of the page to get
     :param articles: Array of articles
     :param articles: String of int of total amount of pages
     """
 
-    for article in articles:
+    def transform_index_article(article):
```

Comment:
Why  is this nested inside `get_index_context`? I don't think it needs to be does it? It's only needs the article that's passed to it right?

If you've no objection, could you please move this function to being top level? (And ideally give it a docstring)